openshift-ci/deploy_metallb.sh

```diff
@@ -62,15 +72,12 @@ find . -type f -name "*clusterserviceversion*.yaml" -exec sed -i 's/quay.io\/ope
 find . -type f -name "*clusterserviceversion*.yaml" -exec sed -r -i 's/name: metallb-operator\..*$/name: metallb-operator.v0.0.0/g' {} +
 
 if [[ "$BGP_TYPE" == "frr-k8s-cno" ]]; then
-awk '/DEPLOY_PODMONITORS/ {system("cat frrk8s-cno.patch"); print; next}1' manifests/stable/metallb-operator.clusterserviceversion.yaml  > temp.yaml
-mv temp.yaml manifests/stable/metallb-operator.clusterserviceversion.yaml
-
-end=$((SECONDS+180))
-oc patch featuregate cluster --type json  -p '[{"op": "add", "path": "/spec/featureSet", "value": TechPreviewNoUpgrade}]'
-while [[ -z $(oc get crds networks.operator.openshift.io -o yaml | grep -i "additionalRouting") ]] && [[ ${SECONDS} -lt ${end} ]]; do
-    sleep 1
-done
+  # - Change the metallb operator's CSV
```

Comment:
Not sure what this comment adds, maybe give a small comment what the not trivial awk is doing under?